src/site/content/en/patterns/index.md

```diff
@@ -1,6 +1,4 @@
 ---
 layout: patterns
 title: Patterns
-description: A collection of code snippets to help you optmize your web projects.
----
-
+description: Core Web Vitals friendly implementations of common UX patterns.
```

Comment:
This description is showing up on the main `/patterns/` page, so this change should probably be reverted.